pkg/kubectl/resource/builder.go

```diff
@@ -60,6 +60,7 @@ type Builder struct {
 
 	singleResourceType bool
 	continueOnError    bool
+	skipFetchingNames  bool
```

Comment:
I agree, but I really wanted the default state (false) to be the normal case (fetch). I couldn't think of a nonnegative name to accomplish that.